bundles/org.openhab.binding.govee/README.md

```diff
@@ -0,0 +1,130 @@
+- H705B Permanent Outdoor Lights 15M
+- H7050 Outdoor Ground Lights 11M
+- H7051 Outdoor Ground Lights 15M
+- H7055 Pathway Light
+- H7060 LED Flood Lights (2-Pack)
+- H7061 LED Flood Lights (4-Pack)
+- H7062 LED Flood Lights (6-Pack)
+- H7065 Outdoor Spot Lights
+- H6051 Aura - Smart Table Lamp
+- H6056 H6056 Flow Plus
+- H6059 RGBWW Night Light for Kids
+- H618F RGBIC LED Strip Lights
+- H618E LED Strip Lights 22m
+- H6168 TV LED Backlight
+
+## Discovery
+
+Discovery is done by scanning the devices in the Thing section. 
+
+The devices _do not_ support the LAN API support out-of-the-box.
+To be able to use the device with the LAN API, the following needs to be done (also see the "Preparations for LAN API Control" section in the [Goveee LAN API Manual](https://app-h5.govee.com/user-manual/wlan-guide)):
+
++ Start the Govee APP and add / discover the device (via Bluetooth) as described by the vendor manual
+Go to the settings page of the device 
+![govee device settings](doc/device-settings.png)
++ Note that it may take several(!) minutes until this setting comes up.
++ Switch on the LAN Control setting.
++ Now the device can be used with openHAB.
++ The easiest way is then to scan the devices via the SCAN button in the thing section of that binding
+
+## Thing Configuration
+
+Even though binding configuration is supported via a thing file it should be noted that the IP address is required and there is no easy way to find out the IP address of the device.
+One possibility is to look for the MAC address in the Govee app and then looking the IP address up via: 
+
+```shell
+arp -a | grep "MAC_ADDRESS"
+```
+
+### `govee-light` Thing Configuration
+
+| Name            | Type    | Description                           | Default | Required | Advanced |
+|-----------------|---------|---------------------------------------|---------|----------|----------|
+| hostname        | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+| macAddress      | text    | MAC address of the device             | N/A     | yes      | no       |
+| deviceType      | text    | The product number of the device      | N/A     | yes      | no       |
+| refreshInterval | integer | Interval the device is polled in sec. | 5       | no       | yes      |
+
+
+## Channels
+
+| Channel               | Type              | Read/Write | Description         |
+|-----------------------|-------------------|------------|---------------------|
+| brightness            | Percentage        | RW         |                     |
+|                       | Switch            | RW         | Power On / OFF      |
+| color                 | Color HSB Type    | RW         |                     |
+| color-temperature     | Percentage        | RW         |                     |
+| color-temperature-abs | Color Temperature | RW         | in 2000-9000 Kelvin |
+
+
+Note: you may have to add "%.0f K" as the state description when creating a color-temperature-abs item.
```

Comment:
It would be even better to define `unit` metadata.

Can you also provide an example for textual configuration like all other bindings have?

Reply:
"It would be even better to define unit metadata."

Not sure how this can be done as I am not familiar with Unit ... and the mix between absolute value and percentage is already a challenge to support in a stable way. I'd rather leave that for later as I feel that a Color Temperature, even though expressed in Kelvin, IMHO to me is not really a "temperature".

Reply:
As for the textual config, I have long stopped using them, so it will take me some time to add that and try it out. Let me improve the documentation later after the release when i have more time, pls.

Reply:
> "It would be even better to define unit metadata."
> 
> Not sure how this can be done as I am not familiar with Unit

```suggestion
Note: you may want to set Unit metadata to "K" when creating a color-temperature-abs item.
```